### DISCUSSION TITLE
Map Preview in Download Screen

### DISCUSSION THREAD

**Discussion by Illuminati-CRAZ** [Quaver/Quaver#4768] [Feature Request]
### Is your feature request related to a problem? Please describe.

I think it'd be nice to be able to get a taste of what the chart is like before downloading the full thing.

### Describe the solution you'd like

Press F3 while a mapset is selected to preview. (But there can be multiple difficulties, which one to preview?)
Send a cropped map file with the audio preview?

**Comment by AiAe** [comment]
Qua files are already accessible directly with download url (/download/map/{map_id}), we can use that + get the preview start time and move the map to there? 
We can go extra mile and remove the non audio preview parts but that might be weird?

**Comment by EpiCreeper** [comment]
i think its really funny when beginner players download a map with a low rating thats apparently a really hard with sv or ln, but i do think this change could be for the best
